From 7d61ad048856ee239870b917d6933e3ad63f0789 Mon Sep 17 00:00:00 2001
From: David King <amigadave@amigadave.com>
Date: Thu, 2 Jul 2015 13:49:22 +0100
Subject: [PATCH] Fix systemd service file

udisks-daemon is installed to $(libexecdir), not $(prefix)/lib/udisks.

https://bugzilla.redhat.com/show_bug.cgi?id=1238664
https://bugs.freedesktop.org/show_bug.cgi?id=91191

Taken-From: [1]
[1] http://pkgs.fedoraproject.org/cgit/udisks.git/tree/udisks-1.0.5-fix-service-file.patch

Upstream-Status: Pending
---
 data/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/Makefile.am b/data/Makefile.am
index 411ea0f..08af5f4 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -31,7 +31,7 @@ systemdservicedir       = $(systemdsystemunitdir)
 systemdservice_DATA     = $(systemdservice_in_files:.service.in=.service)
 
 $(systemdservice_DATA): $(systemdservice_in_files) Makefile
-	@sed -e "s|\@libexecdir\@|$(prefix)/lib/udisks|" $< > $@
+	@sed -e "s|\@libexecdir\@|$(libexecdir)|" $< > $@
 endif
 
 udevrulesdir = $(slashlibdir)/udev/rules.d
-- 
2.4.5
